test/JoeMakerV4.test.ts

```diff
@@ -164,7 +164,7 @@ describe("joeMakerV4", function () {
     it("should convert JOE - AVAX", async function () {
       await this.zap.zapIn(this.joeAvax.address, { value: ethers.utils.parseEther("2") })
       await this.joeAvax.transfer(this.joeMakerV4.address, await this.joeAvax.balanceOf(this.dev.address))
-      await this.joeMakerV4.convert(this.joe.address, this.wavax.address)
+      await this.joeMakerV4.convert(this.joe.address, this.wavax.address, "100")
       expect(await this.usdc.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect(await this.joeAvax.balanceOf(this.joeMakerV4.address)).to.equal(0)
       expect(await this.usdc.balanceOf(BAR_ADDRESS)).to.equal("214466109")
```

Comment:
Again, I think we need better assertions here; the expectation is that Bar receives a certain amount of JOE